Makefile

```diff
@@ -1204,13 +1204,25 @@ lint-addon-docs: test/addons/.docbuildstamp
 cpplint: lint-cpp
 	@echo "Please use lint-cpp instead of cpplint"
 
+.PHONY: lint-python
+# Lints the Python code with flake8.
+# Flag the build if there are Python syntax errors or undefined names
+lint-python: lint-python
+	@echo "Running Python linter on $(shell $(PYTHON) --version)..."
+	@$(PYTHON) -m pip install --upgrade flake8
```

Comment:
This doesn't work on my system when not run as root:
```
OSError: [Errno 13] Permission denied: '/usr/lib/python2.7/site-packages/enum34-1.1.6.dist-info'
make[1]: *** [Makefile:1212: lint-python] Error 2

```

Reply:
Can you please try to add __--user__ to the command and let me know if that works?

Reply:
:ok_hand: It works with `--user`